src/QsCompiler/Tests.Compiler/LinkingTests.fs

```diff
@@ -307,7 +307,7 @@ type LinkingTests (output:ITestOutputHelper) =
                     Assert.Equal(0, ex.TypeParameterResolutions.Count)
             | _ -> ()
 
-        let walker = new TypedExpressionWalker<unit>(new Action<_>(onExpr));
+        let walker = TypedExpressionWalker(Action<_> onExpr, ())
```

Comment:
Hu, I am curious: Do you know what the change is related to new?

Reply:
In F#, `new` is optional unless the type implements `IDisposable`. I always prefer to omit `new`.

Reply:
That makes sense - not a needed change then.